run_all.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
```

Comment:
I converted my upgrade `run_all` shell script to a Python script, so it's easier to modify and could have unit tests written